fdbutil/filter_fdb_entries.py

```diff
@@ -87,15 +87,16 @@ def filter_fdb_entries(fdb_filename, arp_filename, config_db_filename, backup_fi
         Returns:
             None
     """
-    arp_map = get_arp_entries_map(arp_filename, config_db_filename)
+    arp_map, switch_mac = get_arp_entries_map_and_switch_mac(arp_filename, config_db_filename)
 
     with open(fdb_filename, 'r') as fp:
         fdb_entries = json.load(fp)
 
     def filter_fdb_entry(fdb_entry):
         for key, _ in fdb_entry.items():
             if 'FDB_TABLE' in key:
-                return key.split(':')[-1].upper() in arp_map
+                mac = key.split(':')[-1].upper()
+                return mac != switch_mac and mac in arp_map
```

Comment:
if we see mac = switch_mac, it is kind of very strange, can we get a WARNING log here? maybe something is wrong.

Reply:
OK, will add logs